airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/StatusView.tsx

```diff
@@ -79,23 +87,29 @@ const StatusView: React.FC<StatusViewProps> = ({ connection, isStatusUpdating })
       onSubmit: async () => {
         await onReset();
         closeConfirmationModal();
+        setCurrentAction(CurrentActionType.RESET);
       },
       submitButtonDataId: "reset",
     });
   };
 
+  const onSyncNowButtonClick = () => {
+    setCurrentAction(CurrentActionType.SYNC);
+    return startAction({ action: onSync });
+  };
+
   const resetDataBtn = (
-    <Button disabled={isAtLeastOneJobRunningOrPending || isStatusUpdating} onClick={onResetDataButtonClick}>
+    <Button disabled={isStatusUpdating} onClick={onResetDataButtonClick}>
       <FormattedMessage id="connection.resetData" />
     </Button>
   );
 
   const syncNowBtn = (
     <SyncButton
-      disabled={!allowSync || isAtLeastOneJobRunningOrPending || isStatusUpdating}
+      disabled={!allowSync || isStatusUpdating}
       isLoading={isLoading}
       wasActive={showFeedback}
-      onClick={() => startAction({ action: onSync })}
+      onClick={onSyncNowButtonClick}
     >
       {showFeedback ? (
```

Comment:
Because the button is now gone when it starts the sync, it should no longer show feedback the way it does now. Instead, it should continue to show the loading state regardless of isLoading | showFeedback. We need to ask design to provide a place for us to tell the user that the sync has started, the cancel has been completed etc.

Reply:
Thanks! Fixed